src/pmi/PMIDriver.cs

```diff
@@ -161,8 +163,23 @@ private static PrepAllResult Compute(PrepAllInfo pi)
                 p.StartInfo.UseShellExecute = false;
                 p.StartInfo.RedirectStandardOutput = true;
                 p.StartInfo.RedirectStandardError = true;
-                p.StartInfo.FileName = System.Diagnostics.Process.GetCurrentProcess().MainModule.FileName; // spawn ourself; find our full path name.
-                p.StartInfo.Arguments = "PREPALL \"" + pi.assemblyName + "\"" + " " + pi.methodToPrep;
+
+                // Fetch our command line. Split off the arguments.
+#if NETCOREAPP2_1
```

Comment:
Is `NETCOREAPP2_1` magically set in a dotnet CLI compile?

Reply:
MSBuild sets this to facilitate multi-targeting. See for instance: [target frameworks](https://docs.microsoft.com/en-us/dotnet/standard/frameworks).